src/actions/mover.ts

```diff
@@ -43,6 +43,12 @@ export class Mover {
    */
   oldIsDragging: (() => boolean) | null = null;
 
+  /**
+   * The stashed getGesture function, which is replaced at the beginning
+   * of a keyboard drag and reset at the end of a keyboard drag.
+   */
+  oldGetGesture: ((e: PointerEvent) => Gesture | null) | null = null;
```

Comment:
This should have been per-workspace (and patching/unpatching handled carefully) because `Mover` can handle multiple simultaneous moves on one or more workspaces.